test/lib/sdkdriver/src/transport/index.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A {@link Transport} represents an interface for communicating
+ * between the mocha.js process that drives a cross-browser test
+ * and the process in which the test is running.
+ * @interface
+ */
+interface Transport {
+  close(): void;
+  open(): Promise<void>;
+  send(data: any): void;
+}
+
+/**
+ * The {@link Transport} was closed.
+ * @event Transport#close
+ * @param {Error} [error] - Optional Error, if closed unintentionally
+ */
+
+/**
+ * The {@link Transport} received a message.
+ * @event Transport#message
+ * @param {object} data
+ */
```

Comment:
It'd be great to include these as part of the interface, but I'm not sure current best practices on typing EventEmitters.